docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:7.6
+
+### Add dependencies ###
+RUN apt update
```

Comment:
I don't think this is necessary. Node should already have the latest dependencies and our container should be rebuild when node is rebuild. Also this is update not upgrade so it doesn't do a whole lot, though clearing of the update cache and unnecessary packages might be something to consider as it can possibly reduce size of the container. This requires looking into the Dockerfile of node.

Reply:
I don't remember why we added this line in the first place to be honest, but I'm sure it's as old as the project itself. It only adds about 1ms to build time as far as I can tell so I've never questioned it but I'm certainly open to removing it for now and seeing if we run into issues with fresh builds on tests/staging